test/runtests.jl

```diff
@@ -51,7 +51,11 @@ end
     dfrm = DataFrame([categorical(repeat(string.('A':'D'), inner = 6)),
                      categorical(repeat(string.('a':'c'), inner = 2, outer = 4))],
                      [:G, :H])
-    f = @eval(@formula($nothing ~ 1+G*H))
+    if isdefined(StatsModels, :term)
```

Comment:
Better require StatsModels 0.6. and remove the condition.

Reply:
Agreed, no real reason to maintain backward compatibility.  If you do that better also bump the version though right?

Reply:
I dropped support for StatsModels 0.5. Since StatsModels 0.6 requires Julia 1, I removed Julia 0.7 from the list of Julia versions tested on Travis.